modules/account-lib/src/coin/baseCoin/baseTransactionBuilder.ts

```diff
@@ -140,6 +140,34 @@ export abstract class BaseTransactionBuilder {
     return this._coinConfig.name;
   }
 
+  /**
+   * Verified validity windows params if them exist and return a valid validity windows.
+   * @param {ValidityWindow} [params] optional validity windows parameter to validate.
+   * @returns {ValidityWindow} verified validity windows or default values
+   */
+  getValidityWindow?(params: ValidityWindow): ValidityWindow {
+    const size = 100000;
+    const window: ValidityWindow = params;
+
+    window.firstValid = window.firstValid ? window.firstValid : 0;
+    window.lastValid = window.lastValid ? window.lastValid : 0;
+    window.minDuration = window.minDuration ? window.minDuration : size;
+    window.maxDuration = window.maxDuration ? window.maxDuration : size;
+    window.unit = window.unit ? window.unit : 'blockheight';
+    if (window.minDuration > window.maxDuration) {
+      window.maxDuration = window.minDuration;
+    }
```

Comment:
Shouldn't we throw an exception instead? Otherwise we're closing the window here.

Reply:
The reason I'm not throwing an exception is that I am assigning maxDuration to be equal to minDuration by default because maxDuration shouldn't be less than minDuration. I based on [this comment](https://github.com/BitGo/BitGoJS/pull/1793/files#r771034682) for validations.

Reply:
done